samcli/cli/command.py

```diff
@@ -118,3 +127,6 @@ def get_command(self, ctx, cmd_name):
             return None
 
         return mod.cli
+
+    def get_command_packages(self):
```

Comment:
This looks like it is only used for testing. If so, is there no way to get this information through the tests?

Reply:
Removed this one.